src/e2e-test/features/BQExecuter/BQTableCreate.feature

```diff
@@ -0,0 +1,61 @@
+@BQExecute
+Feature: Executes a SQL query in BigQuery Execute
+
+  @BQ_SOURCE_TEST @BQ_SINK_TEST
+  Scenario: Verify BigQuery Execute Query
```

Comment:
Verify data transfer of BigQueryExecute plugin with mandatory fields

Reply:
@sawantpritam -Done